ui/components/GenericModal.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { Modal, Backdrop, Fade } from '@mui/core'
+import { styled } from "@mui/material/styles";
+import { useTheme } from "@mui/system";
+
+const CustomModal = styled(Modal)(({ theme }) => ({
+   display : "flex",
+   alignItems : "center",
+   justifyContent : "center", 
+}))
+/**
+ *
+ * @param {{
+ *  open?: boolean,
+ *  Content?: JSX.Element,
+ *  handleClose?: (event: {}, reason: "backdropClick" | "escapeKeyDown") => void,
+ *  container?: React.ReactInstance | (() => React.ReactInstance)
+ * }} props
+ * @returns
+ */
+
+export const GenericModal = ({
```

Comment:
Can you explain the purpose of GenericModal, in the sense, how is it different from CustomModal ?

Reply:
Sorry, my bad i put in the wrong name here 😅

Reply:
No issues, I actually asked that to make sure that the purpose of that component is well established. If not, we probably don't need it.

Reply:
I thought pf including this as there is a GenericModal in our code base right now.